dotnet-authserver/src/TeacherIdentity.AuthServer/AuthenticationState.cs

```diff
@@ -358,6 +358,7 @@ public void OnMobileNumberVerified(User? user = null)
         }
 
         MobileNumberVerified = true;
+        FirstTimeSignInForEmail = user is null;
```

Comment:
Sign in via exisiting phone found should not show first time sign in complete message